Cargo.toml

```diff
@@ -159,6 +159,11 @@ name = "repl"
 path = "examples/repl.rs"
 required-features = ["help"]
 
+[[example]]
+name = "custom-clap-path-derive"
```

Comment:
This PR is blocked on deciding if we want it, see #4414

(normally, I outright close PRs that are pending discussions)